src/schema/schema_worker.c

```diff
@@ -28,7 +28,12 @@ __wti_execute_handle_operation(WT_SESSION_IMPL *session, const char *uri,
         WT_RET(ret);
     }
 
-    WT_RET(__wt_session_get_btree_ckpt(session, uri, cfg, open_flags, NULL, NULL));
+    if (open_flags & WT_BTREE_VERIFY) {
+        WT_RET(__wt_session_get_btree_ckpt(session, uri, cfg, open_flags, NULL, NULL));
+    } else {
+        WT_RET(__wt_session_get_dhandle(session, uri, NULL, cfg, open_flags));
+    }
```

Comment:
```suggestion
    if (open_flags & WT_BTREE_VERIFY)
        WT_RET(__wt_session_get_btree_ckpt(session, uri, cfg, open_flags, NULL, NULL));
    else
        WT_RET(__wt_session_get_dhandle(session, uri, NULL, cfg, open_flags));
```

Reply:
I will spend more time checking those two functions - thank you for raising this, it's worth digging into.

Reply:
Done